React-Projects/job-portal/src/PostJob.jsx

```diff
@@ -0,0 +1,149 @@
+import React, { useState, useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import "./styles/PostJob.css";
+import axios from "axios";
+
+const PostJob = () => {
+  let isLoggedIn = false;
+  const navigate = useNavigate();
+
+  const jwtCookie = document.cookie
+    .split(";")
+    .find((cookie) => cookie.trim().startsWith("jwt_token="));
+
+  if (jwtCookie) {
+    isLoggedIn = true;
+  }
+
+  const [jobTitle, setJobTitle] = useState("");
+  const [jobType, setJobType] = useState("");
+  const [jobSalary, setJobSalary] = useState(0);
+  const [jobDescription, setJobDescription] = useState("");
+
+  const postAJob = async (event) => {
+    event.preventDefault();
+    var requestBody = {
+      job_name: jobTitle,
+      job_type: jobType,
+      job_description: jobDescription,
+      job_salary: jobSalary,
+    };
+
+    const customConfig = {
+      headers: {
+        "Content-Type": "application/json",
+        Cookie: jwtCookie,
+      },
+    };
+
+    axios.defaults.headers.common.Authorization = `${jwtCookie}`;
+
+    const Client = axios.create({
+      withCredentials: true,
+    });
+
+    const result = await Client.post(
```

Comment:
@codal-kcolaco Please make sure that all the apis are in different file and use them from there